api-client/src/endpoints/createEventsEndpoint.ts

```diff
@@ -49,7 +49,9 @@ export default (baseUrl: string): Endpoint<ParamsType, Array<EventModel>> =>
                 region: event.location.region,
                 country: event.location.country,
                 latitude: event.location.latitude,
-                longitude: event.location.longitude
+                longitude: event.location.longitude,
+                thumbnail: event.thumbnail,
+                path: normalizePath(event.path)
```

Comment:
I think we should not store the same properties twice in the same model. Really confusing and makes you think these are different things when they are not. They also have nothing to do with location.

Reply:
hm well i understand. But you could also say everything that should be shown on the map connected to the poi should be in the geoJSON. So i would have to touch in the frontend again the Feature to merge values

Reply:
Yes, but this is not the geoJSON but just a general model, isn't it? Perhaps you can just extend `convertToPoint` to take these as params and add them to the geoJSON. Did not really think thoroughly about this though.